client/src/components/Genre.jsx

```diff
@@ -0,0 +1,47 @@
+import {
+  List,
+  Datagrid,
+  TextField,
+  Create,
+  SimpleForm,
+  TextInput,
+  Edit,
+  useRecordContext,
+} from "react-admin";
+
+function GenreTitle() {
```

Comment:
this function doesn't get used

Reply:
yes, i was about to use that to show the title, i can remove it